airbyte-cdk/python/build.gradle

```diff
@@ -13,8 +13,8 @@ task generateProtocolClassFilesWithoutLicense(type: Exec) {
     dependsOn ':tools:code-generator:airbyteDocker'
 }
 
-// todo: figure out dependencies
-task generateProtocolClassFiles(dependsOn: [generateProtocolClassFilesWithoutLicense, rootProject.spotlessPython])
-rootProject.spotlessPython.mustRunAfter(generateProtocolClassFilesWithoutLicense)
-rootProject.build.dependsOn generateProtocolClassFiles
-installReqs.dependsOn(generateProtocolClassFiles)
+task generateProtocolClassFiles(dependsOn: [generateProtocolClassFilesWithoutLicense])
```

Comment:
do we still need the distinction between with/without license then? seems like nothing depends on `generateProtocolClassFiles`